Samples/SampleCode/README.md

```diff
@@ -0,0 +1,15 @@
+# vtkU3DExporter
+
+Build and test instructions:
+
+    conda env create
+    conda activate vtku3dexporter
+    mkdir build
+    cd build
+    ccmake ..
```

Comment:
typo?

Reply:
No, that starts an interactive ncurses cmake gui.

Reply:
Actually, ``cmake`` works better because no interaction is actually needed 😉 I'll change it